lib/sycamore/sycamore/tests/unit/transforms/test_union.py

```diff
@@ -0,0 +1,32 @@
+from sycamore.data import Document
+from sycamore.transforms.union import Union
+from sycamore import DocSet
+import sycamore
+
+
+class TestUnion:
+    @staticmethod
+    def setup(exec_mode) -> tuple[list[DocSet], list[list[Document]]]:
+        ctx = sycamore.init(exec_mode=exec_mode)
+        docs_1 = [Document(doc_id="a"), Document(doc_id="b")]
+        docs_2 = [Document(doc_id="c"), Document(doc_id="d")]
+        docs_3 = [Document(doc_id="e"), Document(doc_id="f")]
+        docses = [docs_1, docs_2, docs_3]
+        dses = [ctx.read.document(docs) for docs in docses]
+        return dses, docses
+
+    def test_union_raw(self):
```

Comment:
Up to you, but I'd consider this test redundant with the next one and remove it.